.github/workflows/ci.yml

```
@@ -327,6 +325,26 @@ jobs:
           python make.py python-ci-install
         timeout-minutes: 20
 
+      # We only save the libparsec lib when:
+      # - We are not in a github queue branch (they're a one time use so caching won't help)
+      # - The rust code isn't modifed.
+      # - We haven't add a cache it.
```

"We haven't already cached it" ?

---

Use a suggestion ;)

````markdown
```suggestion
foo bar
```
````

---

done